client/app/views/menu.coffee

```diff
@@ -49,10 +49,26 @@ module.exports = class MenuView extends ViewCollection
     toggleDropdown: ->
         @$('#menuitems').toggleClass 'visible'
 
+    onCalendarMultipleRemove: ->
+        message = t 'confirm delete selected calendars'
+        if confirm(message)
+            @startSpinner()
+            $('.calendar-actions:checked').each ->
+                calendarName = $(this).parent().find('.calendar-name')[0].innerHTML
```

Comment:
Instead of parsing DOM, I recommend that you use the `value` field of your input to get the calendar-name. It'll more future-proof.